.gitignore

```diff
@@ -7,6 +7,8 @@
 /pkg/
 /spec/reports/
 /tmp/
+#Rubymine
```

Comment:
I think it would be great if the specifics to your local setup would stay in a global gitignore instead of the local one for the project